pyposeidon/schism.py

```diff
@@ -677,6 +681,11 @@ def run(self, **kwargs):
 
         proc = tools.execute_schism_mpirun_script(cwd=calc_dir)
 
+        if proc.returncode == 0:
+            # ---------------------------------------------------------------------
+            logger.info("model finished successfully\n")
+            # ---------------------------------------------------------------------
+
```

Comment:
Not a biggie, but would it make sense to move this log message inside `execute_schism_mpirun_script()`? 

I guess the answer depends on whether the function is being used elsewhere

Reply:
I guess, if we put it in tools the logging reference will also be different.

```python
pyposeidon.schism run:671 executing model
pyposeidon.tools :xxx model finished successfully
```

which might be misleading as well, but I am open to it if you think it's better.